src/app/Http/Controllers/QuizController.php

```diff
@@ -5,14 +5,27 @@
 use Illuminate\Http\Request;
 use Illuminate\Http\Response;
 
+use Illuminate\Support\Facades\DB;
+
 class QuizController extends Controller
 {
-    public function quiz1()
+    //問題リスト表示用
+    public function index()
     {
-        return view('quiz.quiz1');
+        // big_questions テーブルのデータを全て取得し $links に格納
+        // getがselectみたいなもん
+        $links = DB::table('big_questions')->get();
+        // list.blade.php で foreach を回し問題リストを表示させる
+        // 階層下がるときは.を使う
+        return view('quiz.list',compact('links'));
+
+        
+        // $choices = DB::table('choices')->get();
+        // return view('quiz/quiz1', ['choices' => $choices]);
     }
-    public function quiz2()
-    {
-        return view('quiz.quiz2');
+    public function quiz($id)
```

Comment:
なんのidかわかりにくくなるためなんのidか明示的な命名の方が良さそうです
```suggestion
    public function quiz($big_question_id)
```